assets/js/common.js

```diff
@@ -658,6 +666,14 @@ var hivepress = {
 				settings['time_24hr'] = true;
 			}
 
+			if (field.data('start-week')) {
```

Comment:
If this one for the time input it's ok to remove it, time inputs have no weekly view, or let me know if I'm missing the use case

Reply:
Also below in the Time field PHP implementation